spec/unit/pushprocessor.spec.ts

```diff
@@ -289,39 +289,6 @@ describe("NotificationService", function () {
         expect(actions.tweaks.highlight).toEqual(true);
     });
 
-    // TODO: This is not spec compliant behaviour.
-    //
-    // See https://spec.matrix.org/v1.5/client-server-api/#conditions-1 which
-    // describes pattern should glob:
-    //
-    // 1. * matches 0 or more characters;
-    // 2. ? matches exactly one character
-    it("should bing on character group ([abc]) bing words.", function () {
```

Comment:
I think we can also remove the `pingpong`, `pies`, and `bakebike` rules from above if these tests are gone?